subsys/suit/platform/sdfw/Kconfig

```diff
@@ -22,6 +22,20 @@ config SUIT_DIGEST_CACHE_SIZE
 
 endif
 
+config SUIT_IPUC
+	bool "Enable in-place updateable components"
+	help
+		SUIT manifest may instruct the platform that component is inactive, by overriding
+		image-size parameter with value 0. Such component may be updated in place,
+		also memory associated to such component may be utilized for other purposes.
```

Comment:
help text indent is 1x tab followed by 2x spaces

Reply:
Fixed!